packages/turf-dissolve/index.js

```diff
@@ -106,8 +106,10 @@ module.exports = function (featureCollection, propertyName) {
             }
             var matchFeature = featureCollection.features[matchFeaturePosition];
 
-            if (matchFeature.properties[propertyName] !== polygon.properties[propertyName]) {
-                continue;
+            if (propertyName !== null) {
```

Comment:
happy to change it to undefined. The existing tests don't pick up any errors even though one of the polys does not have the required property.